### DISCUSSION TITLE
🎧 Kaizen 16 - Just do it - Sep. 20, 2024

### DISCUSSION THREAD

**Discussion by gerhard** [thechangelog/changelog.com#520] [Kaizen]
![badgen.net](https://flat.badgen.net/static/Last%20updated%20on%20September%2021,%202024/PUBLISHED/blue)

[🎧 This is now available as changelog.com/friends/62](https://changelog.com/friends/62) + [📝 episode transcript](https://github.com/thechangelog/transcripts/blob/master/friends/changelog--friends-62.md)

<a href="https://changelog.com/friends/62" target="_blank"><img width="1552" alt="image" src="https://github.com/user-attachments/assets/6a1c93e3-8336-44b7-b802-6713f4d65327"></a>

See all improvements in this 🐙 GitHub repository: [Kaizen 15 ... 16](https://github.com/thechangelog/changelog.com/compare/kaizen.15...kaizen.16)

---
This contains all things related to the next Kaizen episode.

Share your thoughts via comments below:
- Is there something in changelog.com that you, our dear listener, would like us to tackle?
- Is there something that **you** would like to help out with?
- Anything that we - @adamstac @jerodsanto - want to improve?

A rough timeline of what to expect (red line is today):

```mermaid
gantt
    title 🎧 Kaizen 16 timeline
    dateFormat  YYYY-MM-DD
    axisFormat %b %e, %Y

    section CONTRIBUTE
    💻 ALL CONTRIBUTIONS WELCOME :2024-06-28, 77d

    section RECORD

    🎙 Record :milestone, 2024-09-13
    ✂️ Edit :2024-09-13, 7d
    🚀 SHIP IT! :milestone, 2024-09-20

    section NEXT
    Kick-off Kaizen 17 :2024-09-20, 30d
```

---

### What changed? [Kaizen.15 ... Kaizen.16](https://github.com/thechangelog/changelog.com/compare/kaizen.15...kaizen.16)

- ⛑️ https://github.com/thechangelog/pipedream
  - https://github.com/thechangelog/pipedream/pull/1
  - [🎬 Why we started building our single-purpose CDN for changelog.com](https://www.youtube.com/live/ghE59eB465I) - August 16, 2024 - LIVE STREAM
    - ✊ @pmbanugo
- 🧢 Custom feeds
- ⛑️ https://github.com/thechangelog/changelog.com/pull/521
  - ✊ @casey https://github.com/casey/just
- 🎩 Is Neon working for us as expected? State of Neon.
- ⛑️ https://github.com/thechangelog/changelog.com/pull/522
  - https://github.com/thechangelog/changelog.com/pull/523
  - ✊ @hugosantos https://namespace.so/

🎩 @adamstac 🧢 @jerodsanto ⛑️ @gerhard 

---

### Links for the show notes

- https://github.com/thechangelog/changelog.com/discussions/520
- https://github.com/thechangelog/pipedream
- https://www.youtube.com/live/ghE59eB465I
- https://github.com/casey/just
- https://namespace.so/

---

### This is a follow-up to

- https://github.com/thechangelog/changelog.com/discussions/511
- https://github.com/thechangelog/changelog.com/discussions/499
- https://github.com/thechangelog/changelog.com/discussions/485

**Comment by jerodsanto** [comment]
I would like to discuss my work on custom feeds

**Reply by gerhard** [reply]
These are the topics that we have so far:
<img width="992" alt="image" src="https://github.com/user-attachments/assets/5b001ff4-c141-40c7-b505-aa97dc9eccc3">

Anything in particular that you would like to talk about @adamstac ?

**Comment by adamstac** [comment]
I would like to discuss the state of Neon for us and if that's working out as expected.

**Reply by gerhard** [reply]
Great idea! I think that this would be a good place for this topic in the discussion:

<img width="942" alt="image" src="https://github.com/user-attachments/assets/fd997634-403a-4af4-90d0-47982039419b">
